server/server_test.go

```diff
@@ -896,6 +896,24 @@ var _ = Describe("Server package test", func() {
 			testURL("POST", responderPluralURL, adminTokenID, responder, http.StatusCreated)
 		})
 
+		It("Request data not available in context on GET action", func() {
+			testURL("GET", responderPluralURL+"/r1", adminTokenID, nil, http.StatusOK)
+		})
+
+		It("Request data not available in context on DELETE action", func() {
```

Comment:
please add tests for create and update, too

Reply:
other existing tests covers create and update